src/Electrical/Analog/ideal_components.jl

```diff
@@ -28,13 +47,27 @@ function Resistor(;name, R=1.0)
 end
 
 """
-    Capacitor(;name, C=1.0, v_start=0.0) 
+```julia
+function Capacitor(; name, C = 1.0)
+```
+
+Creates an ideal Capacitor.
+
+# States
+- `v(t)`: [`V`]
+  The voltage across the capacitor, given by `D(v) ~ p.i / C`
```

Comment:
The states are actually inherited from the `OnePort` component, hence they are `v` and `i`. Maybe a `@ref` would do it.